### ISSUE TITLE
Review website-revamp for deployment

### ISSUE THREAD

**Issue by locks** [mainmatter/mainmatter.com#910]
Given the substantial amount of work present in the long-lived `website-revamp` branch, a manual review of the entire website should be done before merging and deploying.
The main aspects to keep in mind is that all of the content is updated to the latest prose, and the layouts, shapes and images match the latest designs, both on desktop and mobile since the design is responsive.

## Tasks

- [x] Review content for services pages
- [x] Review links to new services sub-pages: digital products, product design, product development
- Review new shapes
  - [x] Review new header on desktop and mobile
  - [x] Review "Work With Us" shape and photographs on desktop and mobile
  - [x] Review new footer on desktop and mobile
  - [x] Review quote shape on desktop and mobile
  - [x] Review new sidebar shapes and images on desktop and mobile

## Issues found
- [x] "simplabs" must be lowercase everywhere
- [x] Timify case study shape needs new image https://github.com/simplabs/simplabs.github.io/issues/910#issuecomment-596617915
- [ ] Not enough space between client logo and next headline https://github.com/simplabs/simplabs.github.io/issues/910#issuecomment-596615311
- [ ] Too much whitespace between header and shape on homepage on mobile viewport https://github.com/simplabs/simplabs.github.io/issues/910#issuecomment-596616735
- [ ] The inline contact forms should be replaced with images on Team Augmentation and Tutoring pages https://github.com/simplabs/simplabs.github.io/issues/910#issuecomment-596625147

## Resources
- Marketing content: https://3.basecamp.com/3091689/buckets/13652882/vaults/2218312321
- Abstract: https://share.goabstract.com/f110e8e6-5c22-4df6-a991-d2bbfcc63369?sha=97dcd61cb66a6c122274bf5f8f718cf3767f3db7

**Comment by marcoow** [comment]
There's not enough space between the client logos and the next headline on the homepage:

<img width="1073" alt="Bildschirmfoto 2020-03-09 um 16 55 12" src="https://user-images.githubusercontent.com/1510/76232306-d2045500-6226-11ea-81e9-9ffaa22f01d8.png">

I guess we're missing an `offset.after-21`

**Comment by marcoow** [comment]
There's too much whitespace between the header and the shape on the homepage on mobile viewports:

![localhost_4200_](https://user-images.githubusercontent.com/1510/76232431-fc561280-6226-11ea-9566-d70676a5adf8.png)

We're defining `margin-bottom: 10rem` on the header while I think what we want would be using one of the classes from the `offset` block.

**Comment by marcoow** [comment]
We need to be using a different image for the Timify case study shape:

<img width="871" alt="Bildschirmfoto 2020-03-09 um 16 58 10" src="https://user-images.githubusercontent.com/1510/76232600-358e8280-6227-11ea-96cd-89268e5538f6.png">

I think we could use the Timify screenshot we're using in the header, introduced via #860

**Comment by marcoow** [comment]
There is whitespace missing between the 3rd and 4th paragraph of the principles on the services page on mobile viewports:

![localhost_4200_ (1)](https://user-images.githubusercontent.com/1510/76232875-a2a21800-6227-11ea-8343-80be4207e0e9.png)

**Comment by marcoow** [comment]
There is whitespace missing between the _"What we believe about design"_ and _"Your design partner, from “start” to “shipped”"_ sections both on desktop and mobile on the product design page:

![localhost_4200_ (3)](https://user-images.githubusercontent.com/1510/76233132-04628200-6228-11ea-9c46-0dd6b262f09d.png)

**Comment by marcoow** [comment]
There is whitespace missing between the individual sections on the digital products page:

![localhost_4200_ (4)](https://user-images.githubusercontent.com/1510/76233266-3ecc1f00-6228-11ea-87b0-2328e23d7f26.png)

**Comment by marcoow** [comment]
The keyword list under _"Rapid explorations, research and validation"_ should be aligned to the right on the digital products page:

![localhost_4200_ (5)](https://user-images.githubusercontent.com/1510/76233433-7a66e900-6228-11ea-9ae6-e20744ec74d1.png)

**Comment by marcoow** [comment]
There's whitespace missing between the main sections on the product development page:

<img width="1186" alt="Bildschirmfoto 2020-03-09 um 17 09 31" src="https://user-images.githubusercontent.com/1510/76233659-ce71cd80-6228-11ea-97be-486d80bbafe5.png">

**Comment by marcoow** [comment]
_"simplabs"_ must be lowercase everywhere

**Comment by marcoow** [comment]
The inline contact forms should be replaced with images on the _"Team augmentation"_ and _"Tutoring"_ pages

**Comment by marcoow** [comment]
The background element of the header should collapse when there is no background on the header:

<img width="1199" alt="Bildschirmfoto 2020-03-09 um 17 14 41" src="https://user-images.githubusercontent.com/1510/76234092-84d5b280-6229-11ea-971e-a4fcdb8d6c43.png">

**Comment by marcoow** [comment]
We should add a quote component for trainline to be shown under the team augmentation link on the _"Services page"_ with this quote from Carl:

> We started working with simplabs to build a state-of-the-art mobile web app from the ground up and it was an absolute success! Not only are they web specialists, but their expertise in development practices and agile processes made our collaboration an absolute delight – Carl Andersson, General Manager of Trainline international

We can replace the timify quote shape on the services page with that one and move the timify quote to the _"Digital Products"_ page. To not repeat images then, we should replace the Trainline image in the header of the _"Digital Products"_ page with the Timify one.

**Comment by marcoow** [comment]
The keyword lists have a horizontal offset to the left which must be removed:

<img width="672" alt="Bildschirmfoto 2020-03-09 um 17 43 04" src="https://user-images.githubusercontent.com/1510/76236805-8608de80-622d-11ea-896a-9770448852c1.png">

**Comment by marcoow** [comment]
The _"Product Development"_ page is missing this content at the bottom:

> We love open source—we’re the leading Ember.js team in Europe
We co-host EmberFest (the biggest Ember conference in Europe) and we actively contribute to the open source projects we're leveraging for our projects

**Comment by locks** [comment]
> The "Product Development" page is missing this content at the bottom:
This was removed in https://github.com/simplabs/simplabs.github.io/pull/935 as it was not part of the design document. Should it be reverted then?

**Comment by marcoow** [comment]
We should add a quote component for Cardstack with this quote from Ed Faulkner:

> I've had the opportunity to work alongside the simplabs team on both their client-facing work for Cardstack and their open source contributions. They are experts in Ember and they are a team that takes good software-development practice seriously. – Ed Faulkner, Cardstack Lead Developer and member of the Ember.js Steering Committee

That component can then be added to the team augmentation page.

**Comment by marcoow** [comment]
@locks Yes, the design docs are not up to date with the content from basecamp apparently.

**Comment by marcoow** [comment]
done via #947
